bundles/org.openhab.binding.fronius/src/main/java/org/openhab/binding/fronius/internal/handler/FroniusMeterHandler.java

```diff
@@ -69,7 +72,8 @@ public void initialize() {
      */
     @Override
     protected State getValue(String channelId) {
-        if (meterRealtimeBodyData == null) {
+        MeterRealtimeBodyDataDTO localMeterRealtimeBodyData = meterRealtimeBodyData;
+        if (localMeterRealtimeBodyData == null) {
```

Comment:
You code is perfectly fine, so not essential to fix. For keeping changes small you might use a different pattern (scope vs var name). This would prevent all the subsequent renaming in this method obsolete.

This also applies to all the other occurences where a local var was introduced.

```suggestion
        MeterRealtimeBodyDataDTO meterRealtimeBodyData = this.meterRealtimeBodyData;
        if (meterRealtimeBodyData== null) {
```

Reply:
Yeah, but I prefer the more obvious approach with local var names as this tends to be less confusing.